src/js/components/Tabs/index.d.ts

```diff
@@ -20,6 +20,8 @@ export interface TabsProps {
   onActive?: (index: number) => void;
 }
 
-declare const Tabs: React.FC<TabsProps & JSX.IntrinsicElements['div']>;
+export type TabsExtendedProps = TabsProps & JSX.IntrinsicElements['div'];
```

Comment:
Use this as references to solve properties conflict: https://github.com/grommet/grommet/pull/5021/files#diff-37524e012a5eb198c557ffc83a19486994866790935ef45e467d68096560ca90R26